iot_verifier/src/poc.rs

```diff
@@ -802,7 +808,7 @@ mod tests {
 
     const BEACONER_GAIN: u64 = 20;
     const LOC0: u64 = 631615575095659519; // malta
-    const LOC1: u64 = 631615575095699519; // malta but a lil out from LOC0
+    const LOC1: u64 = 631615576056478207; // malta but a lil out from LOC0
```

Comment:
was this value just wrong before?

Reply:
see the note above; this value was previously an invalid cell

Reply:
ah